caravel/controllers/moderation.py

```diff
@@ -58,4 +85,42 @@ def moderation_view():
 
     return render_template("moderation/view.html",
                            inquiries=inquiries,
-                           listings=listings)
+                           listings=listings,
+                           config=config)
+
+
+@app.route("/_internal/automod")
+def automod():
+    if not users.is_current_user_admin():
+        return "???"
+
+    pending = itertools.chain(
+        model.UnapprovedListing().query(),
+        model.UnapprovedInquiry().query(),
+    )
+
+    config = ModerationConfig.get()
+    if not config.enabled:
+        return ""
+
+    for to_moderate in pending:
+        if to_moderate.age < datetime.timedelta(minutes=config.min_delay):
+            continue
+
+        values = dict(to_moderate.to_dict())
+        values["principal"] = values["principal"].email
+        allowed = True
+
+        for field in values.values():
+            for item in config.blacklist:
```

Comment:
Repeated database properties are the empty list by default. I can see where that's not really clear, though. Ideas for better commenting?